src/Views/ProgressView.vala

```diff
@@ -191,11 +191,20 @@ public class ProgressView : AbstractInstallerView {
             value = 0
         };
 
+        // 256 MiB is the minimum distinst ESP partition size, so this is 256 MiB in MB plus a bit
```

Comment:
Just to get this right:

256 MiB ≈ 268,435 MB

So ~10 MB for safety?

Reply:
Approximately yes, since we're working with disk sectors, there will be some amount on the start of the drive we can't use too. So in reality, it'll probably be more like 6-8MB for safety.

I did set it at 256MiB and then distinst failed it because of the reserved sectors on the start of the drive, so I just added 10MB or so 😃

Reply:
Thanks for clarification